modules/peers.js

```diff
@@ -267,11 +267,11 @@ __private.dbLoad = function (cb) {
 			}
 		}, function (err) {
 			library.logger.trace('Peers->dbLoad Peers discovered', {updated: updated, total: rows.length});
-			return setImmediate(cb);
+			return setImmediate(cb, err);
```

Comment:
We will never get `err` here, because we not propagating it from inside `peer.rpc.status`.

Reply:
Will leave it as is then ;)